### PR TITLE
Use monochrome-black trayicon on MacOS

### PR DESCRIPTION
**Pull request by xylix**
Solves https://github.com/ActivityWatch/activitywatch/issues/239 . 
Requires https://github.com/ActivityWatch/media/pull/1

### CODE REVIEW

File: aw_qt/resources.qrc [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <RCC version="1.0">
     <qresource>
         <file alias="logo.png">../media/logo/logo.png</file>
+        <file alias="monochrome-logo.png">../media/logo/black-monochrome-logo.png</file>
```
Comment (ErikBjare) [inline aw_qt/resources.qrc]: Shouldn't you include both the black and white logos here?

File: aw_qt/trayicon.py [hunk 2]
```diff
@@ -166,8 +166,11 @@ def run(manager, testing=False):
         sys.exit(1)
 
     widget = QWidget()
+    if sys.platform == "darwin":
+        icon = QIcon(":/monochrome-logo.png")
```
Comment (ErikBjare) [inline aw_qt/trayicon.py]: And then somehow set the correct color here depending on system theme?

**Comment by johan-bjareholt** [comment]
This library has no non-std dependencies and would solve the detection of a dark/light theme on macOS, could solve the issue nicely
https://github.com/albertosottile/darkdetect

**Comment by xylix** [comment]
Trayicon.py does already import subprocess so a CLI command solution could be applied with no extra imports. But darkdetect leaves the codebase much cleaner and doesn't add unnecessary logic, so it's probably the nicer solution. I'll push a commit in a second.

**Comment by xylix** [comment]
Should the white-monochrome-logo be used on other platforms too or do you think the normal activitywatch logo looks better in Linux / Windows system tray?

**Comment by johan-bjareholt** [comment]
> Should the white-monochrome-logo be used on other platforms too or do you think the normal activitywatch logo looks better in Linux / Windows system tray?

Since we have no idea what kind of theme the user has on the other platforms the safe option would be to use the non-monochrome icon as that doesn't have any transparency.

**Comment by xylix** [comment]
I had some problems getting pyinstaller to package `darkdetect` so I ended up using a Foundation snippet since trayicon.py already imports objc stuff on MacOS anyway. It's quite readable.

**Pull request merged**